sidecar/config/config.go

```diff
@@ -426,3 +450,66 @@ func parseServiceNameAndTags(service string) (name string, tags []string) {
 	}
 	return
 }
+
+type A8ServicesNginxConfData struct {
+	SSLFlag          string
+	CertPath         string
+	CertKeyPath      string
+	CACertPath string
+}
+
+// Proxy nginx configuration values.
+const (
+	A8ServicesConfTemplatePath        = "/etc/nginx/amalgam8-services.conf.tmpl"
+	A8ServicesConfPath                = "/etc/nginx/amalgam8-services.conf"
+	NginxSSLFlag                      = " ssl"
+	NginxSSLCertDirective             = "ssl_certificate "
+	NginxSSLCertKeyDirective          = "ssl_certificate_key "
+	NginxProxySSLTrustedCertDirective = "proxy_ssl_trusted_certificate "
+)
+
+
+// Generate proxy config
+func (c *Config) GenProxyConfig() error {
+    // Generate amalgam8-services.conf 
+    err := c.GenNginxA8ServicesConf()
+    return err
+}
+
+func (c *Config) GenNginxA8ServicesConf() error {
+	var a8ServicesData *A8ServicesNginxConfData
+	logrus.Debug("GenProxyConfig " + strconv.FormatBool(c.Proxy) + " " + c.ProxyConfig.Type)
```

Comment:
Alternate syntax that is a bit easier to read:
`logrus.Debugf("GenProxyConfig %v %v", c.Proxy, c.ProxyConfig.Type)`